src/App.jsx

```diff
@@ -100,10 +100,7 @@ function App() {
               </Routes>
             </Box>
           </Box>
-          <Routes>
-            <Route path="/create-log" />
```

Comment:
put this back we need this because create-log has different footer

Reply:
done